symfony/src/Dto/CustomerDTO.php

```diff
@@ -38,7 +38,7 @@ final class CustomerDTO
     public $email;
     /**
      * @Assert\Type("string")
-     * @Assert\Length(min = 8, max = 20, minMessage = "minimal length is 8 numbers in a string", maxMessage = "maximal length is 20 numbers in a string")
+     * @Assert\Length(min = 8, max = 20)
```

Comment:
disabled message cause actually we have default one and it works as it should (just practiced with asserts)